opt_out_http_api/store/tests/test_memory.py

```diff
@@ -75,8 +75,8 @@ def test_count_one(self):
 
     def test_count_many(self):
         store = OptOutMemory()
-        opt_count_one_many = store.count()
-        self.assertEqual(opt_count_one_many, 0)
+        opt_count_many = store.count()
+        self.assertEqual(opt_count_many, 0)
```

Comment:
I'm not quite sure why this was changed as part of this pull request, but I guess if we're going to change the name of the variable it should be `opt_count` throughout the test?